packages/wallets/src/features/cfd/screens/CreatePassword/CreatePassword.tsx

```diff
@@ -24,9 +29,14 @@ const CreatePassword: React.FC<TProps> = ({ icon, onPasswordChange, onPrimaryCli
                 <input onChange={onPasswordChange} placeholder={`${title} password`} type='password' />
                 <PasswordShowIcon className='wallets-create-password-input-trailing-icon' />
             </div>
-            <button className='wallets-create-password-button' onClick={onPrimaryClick}>
-                Create {title} password
-            </button>
+            {!isMobile && (
+                <WalletButton
+                    disabled={!password}
```

Comment:
Fixed an issue where the button is not disabled when user has not entered any password. In design the button was disabled when input was empty

https://github.com/binary-com/deriv-app/assets/103016120/9e62c6d2-b760-494c-ab3e-81a70eb0d2ce